Resources/doc/usage.rst

```diff
@@ -0,0 +1,180 @@
+Usage
+=====
+
+Add these two lines in your layout:
+
+.. code-block:: twig
```

Comment:
what about using a configuration-block to show the PHP templating example too, for consistency ? converting the snippet should be easy.